homeassistant/components/hassio/manifest.json

```diff
@@ -4,6 +4,7 @@
   "codeowners": ["@home-assistant/supervisor"],
   "dependencies": ["http", "repairs"],
   "documentation": "https://www.home-assistant.io/integrations/hassio",
+  "integration_type": "service",
```

Comment:
This should be changed for the backup integration, not hassio now. 😄

Reply:
damn ... I need more :coffee: :see_no_evil: